openmc/settings.py

```diff
@@ -298,191 +298,191 @@ def __init__(self):
         self._max_tracks = None
 
     @property
-    def run_mode(self):
+    def run_mode(self) -> str:
         return self._run_mode.value
 
     @property
-    def batches(self):
+    def batches(self) -> int:
         return self._batches
 
     @property
-    def generations_per_batch(self):
+    def generations_per_batch(self) -> int:
         return self._generations_per_batch
 
     @property
-    def inactive(self):
+    def inactive(self) -> int:
         return self._inactive
 
     @property
-    def max_lost_particles(self):
+    def max_lost_particles(self) -> int:
         return self._max_lost_particles
 
     @property
-    def rel_max_lost_particles(self):
+    def rel_max_lost_particles(self) -> int:
```

Comment:
Looks like this one is wrong here and elsewhere as well. Can you also fix the docstring above and the type annotation in the setter?
```suggestion
    def rel_max_lost_particles(self) -> float:
```